apps/api/prisma/schema.prisma

```diff
@@ -14,31 +14,131 @@ datasource db {
 }
 
 model TbUsers {
```

Comment:
Acho que "Tb" no início e "s" no final são desnecessários na definição de um modelo de alto nível. Aqui tem alguns exemplos que acho que podemos seguir: https://www.prisma.io/docs/orm/prisma-schema/overview